.github/workflows/build-and-test.yml

```diff
@@ -51,10 +51,31 @@ jobs:
           else
             echo "pr-number=${{ github.event.inputs.pr-number }}" >> $GITHUB_OUTPUT
           fi
-  
+
       - name: Build Docker image
         run: docker build -t ${{ vars.ARTIFACTORY_URL }}/${{ secrets.ARTIFACTORY_REPOSITORY_NAME }}/house-innovations-housnav-web:${{ steps.set-pr-number.outputs.pr-number }} -f apps/web/Dockerfile . --no-cache
 
+      - name: Run Docker container
+        run: docker run -d -p 3000:3000 --name housnav ${{ vars.ARTIFACTORY_URL }}/${{ secrets.ARTIFACTORY_REPOSITORY_NAME }}/house-innovations-housnav-web:${{ steps.set-pr-number.outputs.pr-number }}
+
+      - name: Install Cypress dependencies
+        run: npm install cypress
+        working-directory: ./apps/web
+
+      - name: Run Cypress tests
+        run: npx cypress run --browser chrome
```

Comment:
I suggest we change the default behavior for `npm run e2e` instead of doing a special case here.

Reply:
Changed to `npm run e2e:chrome`